### PR TITLE
ADCM-1844_10 add tests to cluster status, import and action page

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/ui_tests/test_cluster_list_page.py [hunk 1]
```diff
@@ -638,3 +641,86 @@ def test_check_cluster_components_page_add_few_hosts_to_component(
...
+        )
+        cluster_status_page = ClusterStatusPage(app_fs.driver, app_fs.adcm.url, 1).open()
+        group_row = cluster_status_page.get_all_config_groups()[0]
+        cluster_status_page.click_collapse_all_btn()
+        cluster_status_page.wait_group_opened(group_row)
+        cluster_status_page.click_collapse_all_btn()
+        cluster_status_page.wait_group_closed(group_row)
+        cluster_status_page.config.click_on_group(title=SERVICE_NAME)
+        with allure.step("Check group rows"):
+            group_info = cluster_status_page.get_config_group_info(group_row)
```
Comment (skhomuti) [inline tests/ui_tests/test_cluster_list_page.py:670]: maybe you need to check that only one item was found?

File: tests/ui_tests/test_cluster_list_page.py [hunk 2]
```diff
@@ -638,3 +641,86 @@ def test_check_cluster_components_page_add_few_hosts_to_component(
             assert (
                 cluster_components_page.check_that_save_btn_disabled()
             ), "Save button should be disabled"
+
+
+class TestClusterStatusPage:
+    @pytest.mark.usefixtures("create_community_cluster")
+    def test_open_by_tab_cluster_status_page(self, app_fs):
+        cluster_main_page = ClusterMainPage(app_fs.driver, app_fs.adcm.url, 1).open()
```
Comment (skhomuti) [inline tests/ui_tests/test_cluster_list_page.py:649]: probably you should replace `1` with `create_community_cluster.id`

Comment (annademina) [reply tests/ui_tests/test_cluster_list_page.py]: replaced everywhere

File: tests/ui_tests/test_cluster_list_page.py (hunk 1 above)
Comment (annademina) [reply tests/ui_tests/test_cluster_list_page.py:755]: added

**Review by skhomuti — DISMISSED** [review]

**Review by skhomuti — APPROVED** [review]

**Review by Sealwing — APPROVED** [review]

**Pull request merged**